### PR TITLE
[AB#44400] added quick edit to remaining stations

### PR DESCRIPTION
**Pull request by axisuru**
# Pull Request

## Prerequisites

- [ ] The PR is targeting the right branch (`dev` for features and `master` for
      releases)
- [ ] potential **release notes** to the PR description added
- [ ] potential **testing notes** to the PR description added
- [ ] appropriate labels for the PR applied

## Description

This PR adds quick edit to the following stations, 
- Channel Workflows
- Media Licenses
- Ingest

This also refactors the the typings from existing explorer stations.

### CODE REVIEW

File: services/channel/workflows/src/stations/Channels/PlaylistsExplorer/Playlists.tsx [hunk 1]
```diff
@@ -178,6 +184,21 @@ export const Playlists: React.FC = () => {
...
+      quickEditRegistrations={[
+        {
+          component: <PlaylistDetailsQuickEdit />,
+          label: 'Playlist Details',
+        },
+        {
+          component: <ProgramManagementQuickEdit />,
+          label: 'Programs',
+          generateDetailsLink: (item) =>
+            `${routes.generate(routes.playlistDetails, {
```
Comment (axinom-prasad) [inline services/channel/workflows/src/stations/Channels/PlaylistsExplorer/Playlists.tsx]: Could we use `routes.programs` here ?

File: services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx [hunk 2]
```diff
@@ -158,6 +161,22 @@ export const Channels: React.FC = () => {
...
       inlineMenuActions={generateInlineMenuActions}
+      quickEditRegistrations={[
+        {
+          component: <ChannelDetailsQuickEdit />,
+          label: 'Channel Details',
+        },
+        {
+          component: <ChannelImageManagementQuickEdit />,
+          label: 'Manage Logo',
+          generateDetailsLink: (item) => `/channels/${item.id}/logo`,
```
Comment (axinom-prasad) [inline services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx]: I'm not against this approach, but could we use `routes.generate` helper for consistency?

File: services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx [hunk 3]
```diff
@@ -158,6 +161,22 @@ export const Channels: React.FC = () => {
...
+        },
+        {
+          component: <ChannelVideoManagementQuickEdit />,
+          label: 'Manage Videos',
+          generateDetailsLink: (item) => `/channels/${item.id}/videos`,
```
Comment (axinom-prasad) [inline services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx]: same as above

**Review by axinom-prasad — APPROVED** [review]

File: services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx (hunk 2 above)
Comment (axisuru) [reply services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx]: fixed in [1c02f6d](https://github.com/Axinom/mosaic-media-template/pull/384/commits/1c02f6d4910218b3bf1f6d65494fb899e5349822)

File: services/channel/workflows/src/stations/Channels/PlaylistsExplorer/Playlists.tsx (hunk 1 above)
Comment (axisuru) [reply services/channel/workflows/src/stations/Channels/PlaylistsExplorer/Playlists.tsx]: fixed in [1c02f6d](https://github.com/Axinom/mosaic-media-template/pull/384/commits/1c02f6d4910218b3bf1f6d65494fb899e5349822)

File: services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx (hunk 3 above)
Comment (axisuru) [reply services/channel/workflows/src/stations/Channels/ChannelsExplorer/Channels.tsx]: fixed in [1c02f6d](https://github.com/Axinom/mosaic-media-template/pull/384/commits/1c02f6d4910218b3bf1f6d65494fb899e5349822)

**Pull request merged**